pytorch_lightning/strategies/ddp.py

```diff
@@ -490,6 +517,8 @@ def reconciliate_processes(self, trace: str) -> None:
         if len(os.listdir(sync_dir)) == (self.world_size // self.num_nodes):
             return
 
+        if not self._pids:
+            return
```

Comment:
Why add this?

Reply:
Possibility of None iterable yields mypy error